wcfsetup/install/files/lib/form/NotificationUnsubscribeForm.class.php

```diff
@@ -0,0 +1,145 @@
+<?php
+namespace wcf\form;
+
+use wcf\data\user\notification\event\UserNotificationEvent;
+use wcf\data\user\User;
+use wcf\system\exception\IllegalLinkException;
+use wcf\system\request\LinkHandler;
+use wcf\system\WCF;
+use wcf\util\HeaderUtil;
+use wcf\util\StringUtil;
+
+/**
+ * Represents the notification unsubscribe form.
+ *
+ * @author	Tim Duesterhus
+ * @copyright	2001-2020 WoltLab GmbH
+ * @license	GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @package	WoltLabSuite\Core\Form
+ * @since	5.3
+ */
+class NotificationUnsubscribeForm extends AbstractForm {
+	/**
+	 * user to unsubscribe
```

Comment:
```suggestion
```
(Redundant comment.)

Reply:
I wasn't sure about whether attribute contents need to have a description. I'll remember that for the future.

Reply:
We did use them in the past, but there only purpose was for them to be there. Over the time it became clear that most comments are redundant and served no real purpose, which is why we stopped placing comments everywhere. Instead, methods and properties have descriptive names that makes it obsolete to write a comment most of the time.

Once we start using typed properties, parameters and return values, we'll be able to drop even more comments. Also, comments require you to view the specific code, but typing and proper naming enables simple and quick code suggestions.